helper/notifier/notifierguicontrol.pas

```diff
@@ -28,8 +30,7 @@ interface
   TNFormDisappear = (fdpNone, fdpStd, fdpFade, fdpFadeUp, fdpFadeDown,
     fdpUp, fdpDown, fdpUnknown);
   TLabels = array of TLabel;
-  //TButtons = array of TButton;
-  TButtons = array of TSpeedButton;
+  TButtons = array of TATButton;
```

Comment:
The normal TSpeedButton has not the ability to change the style. With the TATButton every aspect of the appearance can be altered.